system/Commands/Database/ShowTableInfo.php

```diff
@@ -0,0 +1,284 @@
+     *
+     * @var array<string, string>
+     */
+    protected $options = [
+        '--show'              => 'Lists the names of all database tables.',
+        '--metadata'          => 'Retrieves list containing field information.',
+        '--desc'              => 'Sorts the table rows in DESC order.',
+        '--limit-rows'        => 'Limits the number of rows. Default: 10.',
+        '--limit-field-value' => 'Limits the length of field values. Default: 15.',
+    ];
+
+    /**
+     * @phpstan-var  list<list<string|int>> Table Data.
+     */
+    private array $tbody;
+
+    private BaseConnection $db;
+
+    /**
+     * @var bool Sort the table rows in DESC order or not.
+     */
+    private bool $sortDesc = false;
+
+    private string $DBPrefix;
+
+    public function run(array $params)
+    {
+        $this->db       = Database::connect();
+        $this->DBPrefix = $this->db->getPrefix();
+
+        $tables = $this->db->listTables();
+
+        if (array_key_exists('desc', $params)) {
+            $this->sortDesc = true;
+        }
+
+        if ($tables === []) {
+            CLI::error('Database has no tables!', 'light_gray', 'red');
+            CLI::newLine();
+
+            return;
+        }
+
+        if (array_key_exists('show', $params)) {
+            $this->showAllTables($tables);
+
+            return;
+        }
+
+        $tableName       = $params[0] ?? null;
+        $limitRows       = (int) ($params['limit-rows'] ?? 10);
+        $limitFieldValue = (int) ($params['limit-field-value'] ?? 15);
+
+        if (! in_array($tableName, $tables, true)) {
+            $tableNameNo = CLI::promptByKey(
+                ['Here is the list of your database tables:', 'Which table do you want to see?'],
+                $tables,
+                'required'
+            );
+
+            $tableName = $tables[$tableNameNo];
+        }
+
+        if (array_key_exists('metadata', $params)) {
+            $this->showFieldMetaData($tableName);
+
+            return;
+        }
+
+        $this->showDataOfTable($tableName, $limitRows, $limitFieldValue);
+    }
+
+    private function removeDBPrefix(): void
+    {
+        $this->db->setPrefix('');
+    }
+
+    private function restoreDBPrefix(): void
+    {
+        $this->db->setPrefix($this->DBPrefix);
+    }
+
+    private function showDataOfTable(string $tableName, int $limitRows, int $limitFieldValue)
+    {
+        CLI::newLine();
+        CLI::write("Data of Table \"{$tableName}\":", 'black', 'yellow');
+        CLI::newLine();
+
+        $this->removeDBPrefix();
+        $thead = $this->db->getFieldNames($tableName);
+        $this->restoreDBPrefix();
+
+        // If there is a field named `id`, sort by it.
+        $sortField = null;
+        if (in_array('id', $thead, true)) {
+            $sortField = 'id';
+        }
+
+        $this->tbody = $this->makeTableRows($tableName, $limitRows, $limitFieldValue, $sortField);
+        CLI::table($this->tbody, $thead);
```

Comment:
@kenjis Since the `--limit-rows 10` is enabled by default, I think creating text to display the **total number of records: 1000** in this section can be useful. and If you do **not have a record**, your table is empty!.
e.g:
```
CLI::write('All table Records:'. function getNumberOfAllRecords(), 'green');
CLI::write('Your table is empty!', 'red');
```

Reply:
You can send another PR after merging this PR.